host-configs/llnl/toss_3_x86_64_ib/intel-oneapi@2021.3.cmake

```diff
@@ -31,6 +31,21 @@ set(ENABLE_FORTRAN ON CACHE BOOL "")
 # fortran compiler
 set(CMAKE_Fortran_COMPILER "${INTELLLVM_HOME}/bin/ifx" CACHE PATH "")
 
+#------------------------------------------------------------------------------
+# Extra flags
+#------------------------------------------------------------------------------
+
+set(BLT_CXX_STD "c++14")
+
+# Use non-system default stdlib
+set(GCC_VERSION "gcc-9.3.1")
+# Note: gcc-8.3.1 and above need the "/rh" directory. Below that version do not.
+set(GCC_HOME "/usr/tce/packages/gcc/${GCC_VERSION}/rh")
+
+set(BLT_C_FLAGS "--gcc-toolchain=${GCC_HOME}" CACHE STRING "")
+set(BLT_CXX_FLAGS "--gcc-toolchain=${GCC_HOME}" CACHE STRING "")
+set(BLT_FORTRAN_FLAGS "-gcc-name=${GCC_HOME}/bin/gcc" CACHE STRING "")
```

Comment:
Interesting this change of flag for the fortran compiler. I’m gonna try that to see if it solves some issue I have. What’s the process that got you there? I mean, did you copy some existing conf, or did you check the compiler options?

Reply:
I see where this comes from: intel is now relying on the llvm compiler, except for frotran. So the options remain the same for ifx, and that should be -gcc-name.
Unfortunately, that did not change anything to the error I have with ifx ;).